deps/verifier/src/tdx/mod.rs

```diff
@@ -20,11 +19,9 @@ pub(crate) mod quote;
 struct TdxEvidence {
     // Base64 encoded CC Eventlog ACPI table
     // refer to https://uefi.org/specs/ACPI/6.5/05_ACPI_Software_Programming_Model.html#cc-event-log-acpi-table.
-    cc_eventlog: Option<String>,
+    eventlog: Option<String>,
```

Comment:
gc did not change this.

Reply:
Oh. I have not commit the fixup change here. Thanks for pointing out.